src/sidebar/editor.js

```diff
@@ -17,22 +17,43 @@ function customizeEditor(editor) {
   });
   
   document.addEventListener('dragover', () => {
+    const theme = browser.storage.local.get(['theme']);
     const mainEditor = document.querySelector('.ck-editor__main');
-    mainEditor.style.border = '1px solid green';
-    mainEditor.style.backgroundColor = '#dcd6da';
+    const editableEditor = document.querySelector('.ck-editor__editable');
+    theme.then(function getTheme(data) {
+      if (data.theme === 'default') {
+        mainEditor.classList.add('drag-n-drop-focus');
```

Comment:
No need for a if anymore. (the all point of the refactoring you did)

Reply:
Actually the thing is that as you can see I have used `mainEditor` and `editableEditor`. For the dark theme simply using `mainEditor` doesn't work as the below line gives `.ck-editor__editable` an extra background color which doesn't allow the changes in `mainEditor` to show.
https://github.com/mozilla/notes/blob/102c8749351311eeb29b5e68b54043a45b57a425/src/sidebar/styles-dark.css#L94
If you want I can remove this background then there would be no need of `if`

Reply:
Yes but you should be able to use css inheritance to merge them nevertheless. I can give it a try if you want.

Reply:
Ok, I'll try this now. If it doesn't work , then for sure you can take over

Reply:
How about now ?